lib/mock/mock-utils.js

```diff
@@ -103,8 +135,7 @@ function safeUrl (path) {
     return path
   }
 
-  const qp = new URLSearchParams(pathSegments.pop())
-  qp.sort()
+  const qp = normalizeQueryParams(pathSegments.pop())
```

Comment:
Can you put this behind an option? This behavior is non-standard.

Reply:
of course, I will look into doing that :slightly_smiling_face::+1: 

my reasoning was that this is more of additive support that allows the other syntaxes without taking anything away from people not caring about/using them, but I guess that it'd be cleaner not to include them by default more from a spec-compliant/correctness point of view? :thinking:

Reply:
I've put the logic in `MockAgent` under a new `acceptNonStandardSearchParameters` option, please have a look and let me know how this looks :pray: 

(PS: I still need to add this to the docs, but before doing that I wanted to check with you to see if I am heading to the right direction :slightly_smiling_face:)